packages/git/node-tests/config-test.js

```diff
@@ -88,7 +88,7 @@ describe('git/config', function() {
 
     env = await createDefaultEnvironment(`${__dirname}/..`, factory.getModels());
 
-    let response = await env.lookup('hub:searchers').search(env.session, 'master', { filter: { type: 'articles' } });
+    let response = await env.lookup('hub:searchers').search(env.session, { filter: { type: 'articles' } });
```

Comment:
@robustdj for the `searchers.search()` we've removed the branch parameter. just delete these.